packages/docs/source/components/tab.md

```diff
@@ -0,0 +1,306 @@
+        <button class="ods-tab" id="tablistitem-1-2" role="tab" aria-selected="false" aria-controls="tab-1-3">Import</button>
+      </div>
+      <div class="ods-tabpanel">
+        <div id="tab-1-0" tabindex="0" role="tabpanel" aria-labelledby="tablistit-1-0" hidden="">
+          <p>Tab Panel 0: Agents</p>
+        </div>
+        <div id="tab-1-1" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-1-1">
+          <p>Tab Panel 1: People</p>
+        </div>
+        <div id="tab-1-2" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-1-2" hidden="">
+          <p>Tab Panel 2: Settings</p>
+        </div>
+        <div id="tab-1-3" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-1-3" hidden="">
+          <p>Tab Panel 3: Import</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</figure>
+
+```html
+<div class="ods-tabs" id="example-1" style="
+  --ods-indicator-width:97px;
+  --ods-indicator-pos-x:103px;
+">
+  <div class="ods-tablist" role="tablist" aria-label="">
+    <button class="ods-tab" id="tablistitem-1-0" role="tab" aria-selected="false" aria-controls="tab-1-0">Agents</button>
+    <button class="ods-tab" id="tablistitem-1-1" role="tab" aria-selected="true" aria-controls="tab-1-1">People</button>
+    <button class="ods-tab" id="tablistitem-1-2" role="tab" aria-selected="false" aria-controls="tab-1-2">Settings</button>
+    <button class="ods-tab" id="tablistitem-1-2" role="tab" aria-selected="false" aria-controls="tab-1-3">Import</button>
+  </div>
+  <div class="ods-tabpanel">
+    <div id="tab-1-0" tabindex="0" role="tabpanel" aria-labelledby="tablistit-1-0" hidden="">
+      <p>Tab Panel 0: Agents</p>
+    </div>
+    <div id="tab-1-1" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-1-1">
+      <p>Tab Panel 1: People</p>
+    </div>
+    <div id="tab-1-2" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-1-2" hidden="">
+      <p>Tab Panel 2: Settings</p>
+    </div>
+    <div id="tab-1-3" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-1-3" hidden="">
+      <p>Tab Panel 3: Import</p>
+    </div>
+  </div>
+</div>
+```
+
+This can be accomplished by changing the following custom properties:
+
+<figure class="ods-table--figure">
+  <table class="ods-table">
+    <caption>Odyssey takes care to ensure apropriate keyboard navigation for the tab component.</caption>
+    <thead>
+      <tr>
+        <th scope="column">CSS variable</th>
+        <th scope="column">Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><code>--ods-indicator-width</code></td>
+        <td>The width of the tab indicator</td>
+      </tr>
+      <tr>
+        <td><code>--ods-indicator-pos-x</code></td>
+        <td>The position of the tab indicator along the x-axis</td>
+      </tr>
+    </tbody>
+  </table>
+</figure>
+
+## Accessibility
+
+### Keyboard support
+<figure class="ods-table--figure">
```

Comment:
We should leave a note here, similar to behavior above, that our package does not ship this behavior - this represents the behavior as intended when implemented.

Also, how does someone escape the Tab component altogether once they're in it?

Reply:
I updated the docs, but I need to go through this again as its sorta unclear. Will update further.

Reply:
Im commenting out Keyboard Support for this release until we have a visual example.